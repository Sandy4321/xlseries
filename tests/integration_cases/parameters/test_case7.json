{"alignment": "horizontal",
 "blank_rows": "True",
 "composed_headers": "False",
 "data_starts": 2,
 "data_ends": 44,
 "frequency": "Y",
 "headers_coord": ["A8", "A10-A12", "A14-A16", "A18-A22", "A24-A26", "A28", "A30", "A32", "A34", "A36-A39", "A41-A44", "A46", "A48", "A50-A52", "A55"],
 "continuity": "False",
 "missings": "False",
 "missing_value": "None",
 "multifrequency": "False",
 "series_names": "None",
 "time_composed": "False",
 "time_alignment": 0,
 "time_multicolumn": "False",
 "time_format": "int",
 "time_header": "False",
 "time_header_coord": "A6",
 "time_header_next_to_data": "False"}
