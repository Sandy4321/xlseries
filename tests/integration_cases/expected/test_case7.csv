,Current Account,Current Account - Goods (7),Current Account - Goods (7) - Exports FOB,Current Account - Goods (7) - Imports FOB,Current Account - Services,Current Account - Services - Exports,Current Account - Services - Imports,Current Account - Income,Current Account - Income - Investment income,Current Account - Income - Investment income - Interests,Current Account - Income - Investment income - Interests - Credit,Current Account - Income - Investment income - Interests - Debit,Current Account - Income - Investment income - Profits and dividends,Current Account - Income - Investment income - Profits and dividends - Credit,Current Account - Income - Investment income - Profits and dividends - Debit,Current Account - Income - Other Income,Current Account - Current Transfers,Capital and Financial Account,Capital and Financial Account - Capital Account,Capital and Financial Account - Financial Account,Capital and Financial Account - Financial Account - Banking Sector,Capital and Financial Account - Financial Account - Banking Sector - Central Bank,Capital and Financial Account - Financial Account - Banking Sector - Other financial entities,Capital and Financial Account - Financial Account - Nonfinancial Public Sector,Capital and Financial Account - Financial Account - Nonfinancial Public Sector - National Government (5) (6),Capital and Financial Account - Financial Account - Nonfinancial Public Sector - Local Governments,Capital and Financial Account - Financial Account - Nonfinancial Public Sector - Companies and other,Capital and Financial Account - Financial Account - Nonfinancial Private Sector,Net Errors and Omissions,International Reserves Variation,International Reserves Variation - BCRA International Reserves,International Reserves Variation - Exchange rate adjustment,Imports CIF
1992-01-01,-5557.75455862,-1395.9,12398.9,13794.8,-2557.4,2984.3,5541.7,-2393.65455862,-2404.65455862,-1480.47729922,2014.00147147,3494.47877069,-924.177259402,335.078219281,1259.25547868,11.0,789.2,9169.07723052,15.7,9153.37723052,1304.7777,288.0,1016.7777,796.0925,1351.7925,43.3,-599.0,7052.50703052,-337.2226719,3274.1,3105.1,-169.0,14981.7
1993-01-01,-8208.88994016,-2363.6,13268.9,15632.5,-3325.64,3070.56,6396.2,-3071.84994016,-3082.54994016,-1581.43570887,2027.27755026,3608.71325913,-1501.11423129,489.101882338,1990.21611363,10.7,552.2,14195.9342226,16.1,14179.8342226,-1694.983,-3064.0,1369.017,7013.6075,6375.6075,879.0,-241.0,8861.20972256,-1736.6442824,4250.4,4480.4,230.0,16872.3
1994-01-01,-10981.4529701,-4138.9,16023.3,20162.2,-3779.0,3364.1,7143.1,-3560.75297009,-3571.25297009,-1771.91531544,3002.312666,4774.22798144,-1799.33765466,459.764641,2259.10229566,10.5,497.2,13781.418149,17.5,13763.918149,1999.305,444.0,1555.305,3994.0595,4367.0595,190.0,-563.0,7770.55364904,-2117.96517894,682.0,561.0,-121.0,21675.1
1995-01-01,-5104.17225429,2357.4,21161.7,18804.3,-3436.1,3825.6,7261.7,-4622.27225429,-4634.57225429,-2540.70707534,3833.87750294,6374.58457828,-2093.86517895,525.102944087,2618.96812304,12.3,596.8,7701.2956945,14.2,7687.0956945,4692.09,1922.0,2770.09,5854.86327414,6324.86327414,394.0,-864.0,-2859.85757964,-2699.12344021,-102.0,-69.0,33.0,20199.7
1996-01-01,-6755.41269419,1759.5,24042.7,22283.2,-3546.94942373,4405.35464249,7952.30406622,-5450.16327046,-5459.96327046,-3379.97271664,3973.12992093,7353.10263756,-2079.99055382,460.726767675,2540.7173215,9.8,482.2,12248.8335,50.8,12198.0335,-197.045,1003.0,-1200.045,9024.01715148,8755.01715148,733.0,-464.0,3371.06134854,-1611.42080584,3882.0,3782.0,-100.0,23855.1
1997-01-01,-12115.5339598,-2122.7,26430.8,28553.5,-4385.11523743,4598.77147136,8983.88670879,-6122.57629425,-6118.97629425,-4199.04115619,4627.4401368,8826.48129299,-1919.93513806,857.241242137,2777.1763802,-3.6,514.857571842,17709.0904679,66.3,17642.7904679,-909.269076923,-586.0,-323.269076923,8162.66549584,6805.66549584,1733.0,-376.0,10389.394049,-2320.5565081,3273.0,3062.0,-211.0,30450.2
1998-01-01,-14464.5980423,-3097.2,26433.7,29530.9,-4444.34797419,4854.06956261,9298.4175368,-7387.11827006,-7382.61827006,-5089.54105095,5257.08042991,10346.6214809,-2293.07721911,868.579545212,3161.65676432,-4.5,464.068201997,18353.6981014,72.9,18280.7981014,3505.20415385,-512.0,4017.20415385,9361.25267045,9525.25267045,148.8,-312.8,5414.34127711,-451.100059154,3438.0,3442.0,4.0,31377.4
1999-01-01,-11909.6446107,-794.6,23308.6,24103.2,-4111.15805584,4718.71872468,8829.87678052,-7457.19661972,-7468.29661972,-5852.49541756,5476.53627813,11329.0316957,-1615.80120216,597.575570703,2213.37677286,11.1,453.31006484,13772.1331043,149.1,13623.0331043,1329.95719407,-1033.0,2362.95719407,10885.9146595,10108.2146595,1364.1,-586.4,1407.16125065,-661.688493558,1200.8,1093.2,-107.6,25508.2
2000-01-01,-8954.51789263,2451.9,26341.0,23889.1,-4283.78347196,4935.52031483,9219.30378679,-7521.67036862,-7529.81402721,-5920.54388824,6431.48413996,12352.0280282,-1609.27013897,978.422629474,2587.69276845,8.14365858988,399.03594795,8731.55648879,105.9,8625.65648879,67.414684892,762.1,-694.685315108,8252.86546308,7408.96546308,1276.4,-432.5,305.376340816,-216.438596161,-439.4,-424.0,15.4,25280.5
2001-01-01,-3780.43524119,7384.9,26542.7,19157.8,-3863.08935634,4627.05307683,8490.14243317,-7726.74070231,-7730.94070231,-7472.70860289,4689.0736819,12161.7822848,-258.232099422,634.769769026,893.001868448,4.2,424.494817451,-5441.54860507,156.5,-5598.04860507,11587.722736,10742.7,845.022736019,-3388.06901144,-2925.61439131,-58.6217821379,-403.832837991,-13797.7023296,-2861.41615374,-12083.4,-12004.6,78.8,20319.6
2002-01-01,8701.65355333,17177.5,25650.6,8473.1,-1460.21978267,3495.40175239,4955.62153507,-7555.63514125,-7530.71838419,-7761.05776613,2653.27966194,10414.3374281,230.339381937,350.365143859,120.025761922,-24.9167570569,540.00847725,-11339.2133225,406.11,-11745.3233225,-2572.65655213,-1807.77699,-764.879562127,3682.87813738,3747.74664766,237.147511773,-302.016022057,-12855.5449078,-1878.08964375,-4515.64941295,-4436.64941295,79.0,8989.5
2003-01-01,8073.07084153,16804.6,29938.8,13134.2,-1193.10164343,4499.91393502,5693.01557845,-8042.53999096,-8017.03792241,-7383.98742995,2615.07607937,9999.06350932,-633.050492455,451.292285908,1084.34277836,-25.5020685572,504.112475917,-3105.20049195,70.055,-3175.25549195,-3000.96585917,-867.845125733,-2133.12073344,4707.85810529,4805.96595206,166.550075625,-264.657922402,-4882.14773806,-1387.09276174,3580.77758784,3642.77758784,62.0,13850.8
2004-01-01,3076.45147219,13264.6,34575.7,21311.1,-1330.99683241,5288.14621882,6619.14305123,-9418.15252539,-9378.92670688,-7091.49198567,2817.68324845,9909.17523412,-2287.4347212,861.272757584,3148.70747879,-39.2258185124,561.000829993,1709.71883238,196.3,1513.41883238,-3097.10748146,-1989.98848146,-1107.119,5109.50080087,5178.39664285,191.274848575,-260.170690557,-498.974487026,532.958329244,5319.12863382,5527.12863382,208.0,22445.3
2005-01-01,5054.9846215,13086.66,40386.8,27300.14,-991.522268959,6634.43479923,7625.95706818,-7524.03373253,-7472.67330758,-3578.07255347,3238.39215937,6816.46471284,-3894.6007541,1022.69557262,4917.29632672,-51.3604249553,483.880622991,3689.95125726,88.9,3601.05125726,-4355.97824667,-2922.02624667,-1433.952,3645.66498156,3562.43342986,334.173600945,-250.942049248,4311.36452236,111.917312524,8856.85319128,8430.85319128,-426.0,28686.94
2006-01-01,7588.1693675,13958.3,46546.2,32587.9,-500.614538516,8022.8590152,8523.47355372,-6328.94827588,-6269.37525271,-1330.07496622,3992.01924104,5322.09420726,-4939.30028649,1637.53006981,6576.8303563,-59.5730231744,459.432181896,-5221.30337834,97.1,-5318.40337834,-10508.0148211,-10399.8828211,-108.132,3616.43247764,3127.90920897,808.879843554,-320.356574883,1573.17896509,1163.28974257,3530.15573172,3960.15573172,430.0,34153.7
2007-01-01,7199.53367673,13455.8,55980.3,42524.5,-512.710746533,10363.1121036,10875.8228501,-6096.98628778,-6025.10221467,-783.973950843,5071.88757768,5855.86152852,-5241.12826383,1487.36924138,6728.49750521,-71.8840731085,353.430711043,5841.09752311,121.097696445,5719.99982666,1583.29030451,1493.17230451,90.118,2564.36148727,2318.44903893,447.899004256,-201.986555911,1572.34803488,57.5230684407,13098.1542683,14139.1542683,1041.0,44707.5
2008-01-01,6636.50208594,15422.6,70018.8,54596.2,-1284.16419711,12155.7815483,13439.9457454,-7671.58910689,-7613.67433597,-1519.70853076,4201.90373939,5721.61227015,-6093.96580521,1323.86598744,7417.83179265,-57.9147709202,169.655389938,-7692.71035972,180.931,-7873.64135972,1732.51173435,1822.59573435,-90.084,-400.473998507,-181.422676961,-102.168401925,-116.882919621,-9205.67909556,1065.09827378,8.89,209.89,201.0,57462.5
2009-01-01,10868.8028259,18525.901,55672.052,37146.151,-1285.34241722,10966.9967204,12252.3391377,-9081.43887468,-9013.66554561,-2386.19482165,2140.31406001,4526.50888166,-6627.47072396,1291.59626362,7919.06698758,-67.773329071,2709.68311779,-9184.55793269,74.028,-9258.58593269,-969.435,-1403.79,434.355,-603.82730035,-503.743808096,-65.4588574392,-34.624634815,-7685.32363234,-338.0348932,1346.21,1581.57,235.36,38786.267
2010-01-01,-1465.9846363,14028.4212171,68187.201,54158.7797829,-1161.15099767,13646.8642522,14808.0152499,-13927.565508,-13865.3980205,-3135.138598,1715.42335512,4850.56195311,-10730.2594225,940.400480572,11670.6599031,-62.1674874722,-405.689347798,7338.84448656,89.035,7249.80948656,-2678.76792601,-2910.0,231.232073986,2907.72754388,1184.76852464,1783.91624236,-60.957223125,7020.8498687,-1715.51985026,4157.34,4222.34,65.0,56792.578074
2011-01-01,-3712.94369919,12925.3204108,84051.130914,71125.8105032,-2246.91760549,15609.9328381,17856.8504436,-13844.0448178,-13789.6393617,-3055.16293,1878.3887297,4933.5516597,-10734.4764317,1225.22108435,11959.697516,-54.4054560836,-547.301686764,365.727103577,62.3226,303.404503577,6899.89324035,5000.0,1899.89324035,-2360.85987707,-3463.80451476,1163.45515791,-60.510520225,-4235.6288597,-2760.67340439,-6107.89,-5813.89,294.0,74319.3485032
2012-01-01,-1176.49685216,15157.7799501,80246.122123,65088.3421729,-3095.07548698,15269.576334,18364.651821,-12758.5053607,-12693.4298597,-3500.40310511,1577.02743361,5077.43053872,-9193.02675463,620.247533358,9813.27428799,-64.7809070834,-480.695954529,1041.24435977,47.845,993.399359768,-1648.19472782,-2000.0,351.805272178,-3338.2016751,-3859.24037694,557.324929713,-36.286227875,5979.79576269,-3169.46750761,-3304.72,-3085.72,219.0,68020.282609
2013-01-01,-4785.70377972,11119.434584,81659.9959069,70540.5613229,-4104.6943156,14767.2127213,18871.9070369,-11011.3164173,-10962.3595094,-3383.97755423,1679.55725108,5063.53480531,-7578.38195518,817.714278915,8396.0962341,-48.9569078463,-789.132757684,-1316.1668774,33.333503,-1349.5003804,-1154.8691892,-2000.0,845.1308108,574.265722459,605.813170183,-145.140620306,113.593172581,-768.896913654,-5721.89421604,-11823.77,-12689.77,-866.0,73655.536673
